src/CoreWCF.Primitives/src/CoreWCF/Channels/SecurityBindingElement.cs

```diff
@@ -298,6 +299,240 @@ private static BindingContext CreateIssuerBindingContextForNegotiation(BindingCo
             }
         }
 
+        // If any changes are made to this method, please make sure that they are
+        // reflected in the corresponding IsIssuedTokenOverTransportBinding() method.
```

Comment:
Comment isn't needed as IsIssuedTokenOverTransportBinding is for client WSDL import

Reply:
You still left the comment, please cleanup